cni/network/network.go

```diff
@@ -801,13 +875,15 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 
 		epInfos = append(epInfos,
 			&network.EndpointInfo{
+				IfName:            epInfo.IfName,
```

Comment:
check if this is set when parsing secondaryInterfaceInfo

Reply:
yes